chore: align README for non-prefixed packages (#2994)
vaadin/web-components: packages/list-box/README.md

# @vaadin/list-box

> A web component allowing to select one or more values from a scrollable list of items.

Reviewer: ```suggestion
A web component for selecting one or more values from a scrollable list of items.
```

Author: Done.
